index.html

```diff
@@ -0,0 +1,157 @@
+        </div>
+    </div>
+
+    <!---Main-->
+    <div class="container">
+        <div class="banner">
+            <img src="images/banner.png">
+        </div>
+        <div class="list-container">
+            <div class="vid-list">
+                <a href=""><img src="images/thumbnail1.png" class="thumbnail"></a>                
+                <div class = "flex-div">
+                    <img src="images/Jack.png">
+                    <div class="vid-info">
+                        <a href="">Best channel to learn coding</a>
+                        <p>Adorer Tutorials</p>
+                        <p>15k Views &bull; 5 days</p>
+                    </div>
+                </div>
+            </div>
+            <div class="vid-list">
+                <a href=""><img src="images/thumbnail2.png" class="thumbnail"></a>                
+                <div class = "flex-div">
+                    <img src="images/Jack.png">
+                    <div class="vid-info">
+                        <a href="">Best channel to learn coding</a>
+                        <p>Adorer Tutorials</p>
+                        <p>15k Views &bull; 5 days</p>
+                    </div>
+                </div>
+            </div>
+            <div class="vid-list">
+                <a href=""><img src="images/thumbnail3.png" class="thumbnail"></a>                
+                <div class = "flex-div">
+                    <img src="images/Jack.png">
+                    <div class="vid-info">
+                        <a href="">Best channel to learn coding</a>
+                        <p>Adorer Tutorials</p>
+                        <p>15k Views &bull; 5 days</p>
+                    </div>
+                </div>
+            </div>
+            <div class="vid-list">
+                <a href=""><img src="images/thumbnail4.png" class="thumbnail"></a>                
+                <div class = "flex-div">
+                    <img src="images/Jack.png">
+                    <div class="vid-info">
+                        <a href="">Best channel to learn coding</a>
+                        <p>Adorer Tutorials</p>
+                        <p>15k Views &bull; 5 days</p>
+                    </div>
+                </div>
+            </div>
+            <div class="vid-list">
+                <a href=""><img src="images/thumbnail5.png" class="thumbnail"></a>                
+                <div class = "flex-div">
+                    <img src="images/Jack.png">
+                    <div class="vid-info">
+                        <a href="">Best channel to learn coding</a>
+                        <p>Adorer Tutorials</p>
+                        <p>15k Views &bull; 5 days</p>
+                    </div>
+                </div>
+            </div>
+            <div class="vid-list">
+                <a href=""><img src="images/thumbnail6.png" class="thumbnail"></a>                
```

Comment:
I think you can add a dead link to your a tag until you are ready to link it to the correct page, this makes it have a clickable link.